backend/server/index.js

```diff
@@ -10,7 +10,7 @@ const expressSession = require('express-session')
 // commented out for now
 // const bodyParser = require('body-parser')
 
-//TODO: this throws a huge error because
+//TODO: this throws a huge error because mongoose is not hooked up yet, please resolve
```

Comment:
can you be more specific here so they know exactly what they have to do

Reply:
changed to become more clear, nice catch